sycl/include/CL/sycl/intel/sub_group_host.hpp

```diff
@@ -18,9 +18,40 @@ namespace cl {
 namespace sycl {
 template <typename T, access::address_space Space> class multi_ptr;
 namespace intel {
-struct minimum {};
-struct maximum {};
-struct plus {};
+
+template <typename T = void> struct minimum {
```

Comment:
Is `minimum`, `maximum` and `plus` definitions in `sub_group_host.hpp` anyhow different from the `sub_group.hpp` definitions?
If no, can we take them into a common header file?

Reply:
Good idea.  Moved to `sycl/intel/functional.hpp` in 9b03fec8356837ea26a0197e5c69fc970885772a .